vowpalwabbit/core/src/reductions/details/automl_impl.h

```diff
@@ -27,13 +27,15 @@ constexpr uint64_t CONFIGS_PER_CHAMP_CHANGE = 10;
 
 using interaction_vec_t = std::vector<std::vector<namespace_index>>;
 
-struct aml_estimator : VW::estimator_config
+template <typename estimator_impl>
+struct aml_estimator
 {
-  aml_estimator() : VW::estimator_config() {}
-  aml_estimator(double alpha, double tau) : VW::estimator_config(alpha, tau) {}
+  estimator_impl _estimator;
```

Comment:
nit: why the underscore in the name?

Reply:
intent for it to be private member bar but right now it is not.